src/component/playground/Playground/PlaygroundForm/PlaygroundCodeFieldset/PlaygroundEditor/PlaygroundEditor.tsx

```diff
@@ -0,0 +1,133 @@
+import CodeMirror from '@uiw/react-codemirror';
+import { json } from '@codemirror/lang-json';
+import { Dispatch, SetStateAction, VFC, useCallback } from 'react';
+import { styled, useTheme, Box } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import Check from '@mui/icons-material/Check';
+import { Error } from '@mui/icons-material';
+
+interface IPlaygroundEditorProps {
+    context: string | undefined;
+    setContext: Dispatch<SetStateAction<string | undefined>>;
+    error: string | undefined;
+}
+
+const StyledEditorHeader = styled('aside')(({ theme }) => ({
+    height: '50px',
+    backgroundColor: theme.palette.grey[100],
+    borderTopRightRadius: '5px',
```

Comment:
border radius has values in `theme`
(`theme.shape.`, borderRadius, borderRadiusMedium, borderRadiusLarge, borderRadiusExtraLarge)

Reply:
Thanks!